src/list.json

```diff
@@ -222,7 +222,18 @@
         "link": "https://github.com/pusher/hacktoberfest/blob/main/README.md"
       }
     ],
-    "Q": [],
+    "Q": [
+      {
+        "organization": "QuestDB",
```

Comment:
Need to Re-style by pasting the command in the Repo directory.
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2337006/patch | git am`
`git push`